src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,20 @@
+import { useLocation, useMatches } from '@remix-run/react';
+import { RouteMatch } from '~/router/types';
+import styles from './breadcrumbs.module.scss';
+import { ArrowRightIcon } from '../icons';
+
+export const Breadcrumbs = () => {
+    const matches = useMatches() as RouteMatch[];
+    const location = useLocation();
```

Comment:
Will we be able to create a board for this component?

Reply:
Hope so. If not, I will create a dumb version.